go/obscuronode/enclave/evm/evm_facade.go

```diff
@@ -19,19 +19,20 @@ import (
 
 // ExecuteTransactions
 // header - the header of the rollup where this transaction will be included
-func ExecuteTransactions(txs []nodecommon.L2Tx, s *state.StateDB, header *nodecommon.Header, rollupResolver db.RollupResolver, chainID int64) map[common.Hash]*types.Receipt {
+// startAtCount - for the receipts and events, the evm needs to know for each transaction the order in which it was execute in the block.
```

Comment:
Found this param name/desc a bit confusing, we could maybe call it txIndex to match the naming in geth?

Reply:
fromTxIndex sounds better.